tests/nested/manual/core-early-config/task.yaml

```diff
@@ -61,3 +61,7 @@ execute: |
     # early config is witnessed by install hook of the pc gadget
     execute_remote "cat /var/snap/pc/common/debug.txt" | MATCH "/dev/null"
     execute_remote "test -L /etc/systemd/system/rsyslog.service"
+
+    # timezone is set
+    execute_remote "cat /etc/timezone" | MATCH "Europe/Malta"
+    execute_remote "readlink -f /etc/localtime" | MATCH "Europe/Malta"
```

Comment:
A better check would be to use install hook of this test to inspect /etc/localtime symlink, similiar to what it does for rsyslog above. This will ensure it was applied early.